core/modules/widgets/link.js

```diff
@@ -153,7 +153,7 @@ LinkWidget.prototype.handleDragStartEvent = function(event) {
 			}
 			// Then the data
 			dataTransfer.clearData();
-			var jsonData = this.wiki.getTiddlerAsJson(this.to),
+			var jsonData = this.wiki.getTiddlerAsJson(this.to,true),
```

Comment:
So, we first generate the json and then inject the **_origin** field... can do that, I was just thinking that this **_origin** thing may just end up being used in exactly this manner elsewhere.